website/docs/d/continuous_compliance_notification.html.markdown

```diff
@@ -105,7 +105,13 @@ at least one of  `alerts_console`, `scheduled_report`, or `change_detection` mus
       * `auth_method` - authentication method; "NoAuth" by default
       * `username` - username in endpoint system
       * `password` - password in endpoint system
-      * `format_type` - format for JSON block for finding; can be "Basic" or "ServiceNow"
+      * `format_type` - format for JSON block for finding, "JsonWithFullEntity" by default; can be one of: "JsonWithFullEntity", "SplunkBasic", "ServiceNow", "QRadar", "JsonFirstLevelEntity", "Jira"
```

Comment:
I think we dont need to list all the option in the description, as it is documented below